eng/config/PublishData.json

```
@@ -210,29 +210,21 @@
       "nugetKind": [ "Shipping", "NonShipping" ],
       "version": "3.9.*",
       "packageFeeds": "default",
-      "channels": [ "dev16.9p3" ],
+      "channels": [ "dev16.9p4" ],
```

As Sam intuited, this is missing the "dev16.9".

```suggestion
      "channels": [ "dev16.9", "dev16.9p4" ],
```